specification/communication/data-plane/Identity/stable/2023-08-01/examples/CreateIdentity.json

```diff
@@ -0,0 +1,30 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01",
+    "content-type": "application/json",
+    "endpoint": "https://my-resource.communication.azure.com",
+    "headers": {
+      "Repeatability-Request-ID": "4e7d453c-ae97-4778-ad51-cb3b4a701100",
+      "Repeatability-First-Sent": "Sun, 10 Sep 2023 20:39:30 GMT"
+    },
+    "body": {
+      "createTokenWithScopes": [
+        "chat"
+      ],
+      "expiresInMinutes": 60
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "identity": {
+          "id": "8:acs:2dee53b4-368b-45b4-ab52-8493fb117652_00000005-14a2-493b-8a72-5a3a0d000081"
+        },
+        "accessToken": {
+          "token": "token",
+          "expiresOn": "2023-09-10T21:39:39.3244584+00:00"
+        }
+      }
+    }
+  }
+}
```

Comment:
also, add the `Repeatability-Result` pls (similarly to [email](https://github.com/Azure/azure-rest-api-specs/search?q=%22Repeatability-Result%22)). 

btw - is it possible to add it also to the swagger, not only the examples?

Reply:
Done.